src/pybind/mgr/volumes/module.py

```diff
@@ -591,6 +590,25 @@ def config_notify(self):
                     elif opt['name'] == "snapshot_clone_no_wait":
                         self.vc.cloner.reconfigure_reject_clones(self.snapshot_clone_no_wait)
 
+    def _is_valid_name(self, name):
```

Comment:
`_is_name_valid()`  is better name/more readable IMO since it's more grammatically correct.

```suggestion
    def _is_name_valid(self, name):
```

Reply:
renamed to `_is_name_valid(self, name)`